### PR TITLE
Add spanish translation for-meeting-notes and the _index.es.md file is added

### PR DESCRIPTION
**Pull request by BlueeeMoon**

### CODE REVIEW

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+2017-09-05 - 2017-09-07
+
+coredev.tech septiembre 2017
+
+<https://twitter.com/kanzure/status/907233490919464960>
+
+((Como siempre, cualquier error es probablemente mío, etc.))
+
+<div id="intro" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:17]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+# Introducción
+
+Existe una gran preocupación sobre si BlueMatt se ha convertido en un nombre erróneo.
+
+Presentación del lunes por la noche: <http://diyhpl.us/wiki/transcripts/sf-bitcoin-meetup/2017-09-04-jonas-schenlli-bip150-bip151/>
+
+Creo que deberíamos seguir usando #bitcoin-core-dev para cualquier cosa relacionada con el cambio de Bitcoin Core y tratar de mantener las cosas abiertas aunque estemos juntos aquí hoy y mañana y el siguiente.
+
+<div id="rescans" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:27]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+No debería discriminar a los usuarios menos sofisticados: digamos que su transacción puede ser más barata a cambio de algo de privacidad; tenemos esta mejora masiva de rendimiento con dbcache que sólo los usuarios sofisticados conocen. Este es un argumento en contra de tener configuraciones. Bueno, hay una opción de GUI para el tamaño de dbcache, pero está escondida en los menús.
+
+Entonces, ¿vamos a añadir una opción GUI para qué tipo de dirección de cambio?
+
+P2SH ... testigo. Por defecto para el cambio a partir de uno es testigo desnudo, pero eso todavía está abierto a debate. Si nadie lo debate, entonces no cambia. Bueno, depende de quien lo implemente. Joinmarket quiere usar Bech32, openbazaar lo está usando, ... Muy bien, a comer.
+
+<div id="p2sh-adopcion" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:143]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+
+# Adopción de P2SH
+
+¿Cuánto tiempo pasó con P2SH antes de que se pudiera asumir en general que se podía enviar? Bueno, bastante tiempo. No se puede... tardó años, pero blockchain.info tiene soporte para P2SH ahora, sí. Todavía lo tienen. Así que ahora mismo si envías a un... no lo indexan, no puedes ver las transacciones de una determinada dirección, aparece como... ¿así que todo el mundo los usa porque son "privados"? Llevó años... quizás más rápido por estas fechas, pero aún así algún tiempo.
+
+blockchain.info no mantuvo su sitio durante unos 2 años. Armory tampoco lo implementó durante mucho tiempo. También hay casos especiales, como decir que tienes una cartera de Bitcoin Core. Coinbase era... bech32... digamos que tengo un exchange... el caso particular de enviarse a uno mismo desde un servicio es un caso que la gente utilizará rápidamente. Nadie quiere hacer un viaje de ida y vuelta extra. Si me pago a mí mismo, lo acepto. Alguien me ha preguntado si podemos tener el URI del bip.. y le he dicho que el problema es que son más cosas de las que preocuparse.
+
+<div id="testnet" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:151]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+
+# Algoritmo de ajuste de la dificultad de testnet
+
+El algoritmo de ajuste de la dificultad de testnet es la razón por la que la altura del bloque es alta en testnet3.
+
+<div id="bumpfee" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:157]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+
+# bumpfee y retransmisión
+
+¿Los monederos suelen retransmitir si la transacción no está en la blockchain después de unos días? Anyonecanspend covenant utxo que necesita para crear uno nuevo de sí mismo en su salida. Si tuvieras covenants, podrías de alguna manera .... oh podríamos hacerlo de esa otra manera, pero no es tan genial como los covenants. La protección de repetición con pactos sería un buen uso de los pactos en lugar de algunos otros usos de los pactos. Segwit2x debería añadir una fuerte protección de repetición bidireccional.
+
+<div id="libevent" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:163]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+
+La implementación anterior era un solo hilo para la red y los compañeros se conectaron a la secuencia con una pausa en el medio. Pero ahora con el bucle de eventos podemos hacer múltiples intentos de conexión simultáneos y hacerlo todo a la vez de forma asíncrona.
+
+El nuevo plan para Cory es que fusionaremos su trabajo inmediatamente, luego sólo haremos que Cory arregle los problemas a medida que aparezcan, y cambiaremos el nombre de Bitcoin Core a Bitcoin Cory.
+
+<div id="toolchain-builder" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:175]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+
+# otros
+
+Si cambias todas las transacciones del historial para usar índices, es un 30% más de reducción en el tamaño de la cadena. Así que obtienes una reducción del 20% usando... pero si cambias todas las... es otro 30% de reducción. Es un ahorro realmente bueno. Tienes que tener un índice de...
+
+# otro otro
+
+<http://bitcoin.sipa.be/bech32/demo/demo.html>
+
+<div id="signature-aggregation" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:193]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+
+Tenemos muchas cosas diferentes en la métrica. Ahora mismo sólo priorizamos estrictamente las cosas. Ya tenemos 6 cosas de confirmación, luego 1 cosa de confirmación, luego 0 cosas de confirmación, y tal vez queremos tratar cada dirección como una unidad, entonces vas a .... No estoy seguro de que discardfee. Tal vez minimizar discardfee. No, sólo lo que sea. Podríamos minimizar el número de entradas, minimizar la cantidad de cuota que descartamos....
+
+<div id="release-validation" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:247]: This `<div/>` is not needed

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+
+# Liberación de la validación
+
+Tengo una pubkey en el binario. Una herramienta de descarga para comprobar el binario que has descargado, como alternativa a pedir al usuario que compruebe manualmente que la liberación ha sido firmada por los desarrolladores de bitcoin. verifysignature no sirve para esto. sha256. Sólo digo que el trabajo mínimo. Eso sería un pequeño número de líneas.
+
+<div id="jenkins" />
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md:253]: This `<div/>` is not needed

**Review by kouloumos — COMMENTED** [review]
There are some HTML elements that are probably from the [original transcript](http://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/2017-09-05/) and are not needed anymore. I've highlighted them:

File: bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md (hunk 1 above)
Comment (BlueeeMoon) [reply bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md]: I have removed the **HTML** elements, thank you.

**Comment by kouloumos** [comment]
Thank you! 🚀

**Pull request merged**